extras/rya.export/export.accumulo/pom.xml

```diff
@@ -102,6 +102,24 @@ under the License.
                     <outputDirectory>src/gen/java</outputDirectory>
                 </configuration>
             </plugin>
+
```

Comment:
Plugin execution is unnecessary if code is stored in ${project.build.directory}/generated-sources.